src/web/trykkomania/trykkomania.md

```diff
@@ -0,0 +1,299 @@
+
+  ```js
+  el.style.top = tilfeldig()
+  el.style.left = tilfeldig()
+  ```
+
+- Dette er det samme som om vi hadde skrevet:
+
+  ```js
+  el.style.top = Math.random * 100 + '%'
+  el.style.left = Math.random * 100 + '%'
+  ```
+
+  Derfor laget vi `tilfeldig`, slik at vi koden blir enklere å lese.
+
+- Hvis du nå trykke på knappen <button>Run</button>, flytter ballen på seg?
+
+  ![](tilfeldig_plassering.gif)
+
+- Trykker du nok antall ganger, legger du merke til at ballen noen gang kommer utenfor bunnen og høyre side.
+- For å unngå dette kan vi begrense `tilfeldig`:
+
+  ```js
+  function tilfeldig () {
+    return Math.random() * 80 + '%'
+  }
+  ```
+
+- Siden `Math.random()` maks er 1 og vi ganger med 80, vil aldri ballen flytte seg lenger ut enn 80% fra toppen eller venstre side.
+
+# Steg 5: Flytte ballen hvert sekund {.activity}
+Vi ønsker at ballen hele tiden skal flytte seg. Nå skal vi bruke `setInterval` til å flytte ballen hvert andre sekund.
+
+## Sjekkliste {.check}
+- Flytt `el.style.left` og `el.style.top` inni en funksjon vi kaller `tegn`:
+
+  ```js
+  function Ball() {
+    var el = document.createElement('div')
+    el.style.backgroundColor = 'black'
+    el.style.width = '60px'
+    el.style.height = '60px'
+    el.style.borderRadius = '30px'
+    el.style.position = 'fixed'
+    document.body.appendChild(el)
+
+    function tegn () {
+      el.style.top = tilfeldig()
+      el.style.left = tilfeldig()
+    }
+  }
+  ```
+
+- Vi kan nå bruke `tegn` for å flytte ballen til et tilfeldig sted.
+- La oss bruke `tegn` med `setInterval`. Legg denne linjen etter `tegn`-funksjonen:
+
+  ```js
+  setInterval(tegn, 2000)
+  ```
+
+- `setInterval(tegn, 2000)` betyr kjør `tegn` hvert `2000` millisekund. 2000 millisekund er 2 sekunder, altså tegnes ballen på et nytt sted hvert andre sekund.
+
+# Steg 6: Poeng {.activity}
+Nå har du en ball som spretter rundt. La oss legge til poeng. Vi har laget en poeng-komponent som du kan bruke, så kommer du raskere i gang. `Poeng` er bygd opp likt som `Ball`, så du må gjerne lese gjennom og se om du forstår den.
+
+## Sjekkliste {.check}
+- Legg til `Poeng` koden:
+
+  ```js
+  /**
+  * Poeng - viser poengsum nede i venstre hjørne.
+   *
+   * Bruk:
+   *   var poeng = Poeng()
+   *   poeng.ink()  // øker poengsummen med 100
+   *   poeng.nullstill()  // setter poengsummen til 0
+   */
+  function Poeng () {
+    var el = document.createElement('div')
+    el.style.position = 'fixed'
+    el.style.bottom = '5px'
+    el.style.left = '8px'
+    el.style.padding = '5px'
+    el.style.backgroundColor = 'black'
+    el.style.color = 'white'
+
+    var poeng = 0
+    el.innerHTML = poeng + ' poeng'
+    document.body.appendChild(el)
+
+    function ink () {
+      poeng += 100
+      el.innerHTML = poeng + ' poeng'
+    }
+    function nullstill () {
+      poeng = 0
+      el.innerHTML = poeng + ' poeng'
+    }
+
+    return { ink: ink, nullstill: nullstill }
```

Comment:
Dersom vi ikke bruker nullstill, kunne vi kanskje bare returnert funksjonen ink, istedet for et objekt?